README.md

```diff
@@ -622,6 +622,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Crystal Okedi](https://github.com/crystalokd)
   - [Cux Sabrina](https://github.com/theflucs)
   - [Cynthia Teeters](https://github.com/cynthiateeters)
+  - [Chiranjeev Thomas](https://github.com/code0monkey1)
```

Comment:
Your name must be in alphabetical order: in line 609, after Chirag Gupta.
Let us know if you need any help with updating that. 🙂